markdown-mode.el

```diff
@@ -8174,6 +8229,33 @@ Translate filenames using `markdown-filename-translate-function'."
           (markdown--browse-url (markdown-link-url)))
     (user-error "Point is not at a Markdown link or URL")))
 
+(defconst markup-props (list 'invisible 'markdown-markup
```

Comment:
Please add package prefix to global variables according to Emacs lisp naming convention

Reply:
I fixed this and ran a linter to help catch some other issues